docker/iceberg-rest-adapter-image/README.md

```diff
@@ -0,0 +1,87 @@
+<!--
+  - Licensed to the Apache Software Foundation (ASF) under one
+  - or more contributor license agreements.  See the NOTICE file
+  - distributed with this work for additional information
+  - regarding copyright ownership.  The ASF licenses this file
+  - to you under the Apache License, Version 2.0 (the
+  - "License"); you may not use this file except in compliance
+  - with the License.  You may obtain a copy of the License at
+  -
+  -   http://www.apache.org/licenses/LICENSE-2.0
+  -
+  - Unless required by applicable law or agreed to in writing,
+  - software distributed under the License is distributed on an
+  - "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  - KIND, either express or implied.  See the License for the
+  - specific language governing permissions and limitations
+  - under the License.
+  -->
+
+# Iceberg rest adapter image
```

Comment:
```suggestion
# Iceberg Technology Compatibility Kit
```

Reply:
> A Technology Compatibility Kit (TCK) is a suite of tests that at least nominally checks a particular alleged implementation of a Java Specification Request ...

To call it as TCK, it should have test suites. This Docker (runtime jar) doesn't pack the test cases. It only has test fixture. 

Maybe we can rename to `Iceberg REST Catalog Adapter Test Fixture`